src/containers/shared/components/Transaction/OracleSet/Simple.tsx

```diff
@@ -0,0 +1,84 @@
+import { useTranslation } from 'react-i18next'
+import { TransactionSimpleComponent, TransactionSimpleProps } from '../types'
+import { SimpleRow } from '../SimpleRow'
+import { SimpleGroup } from '../SimpleGroup'
+import { MILLIS_PER_SECOND } from '../../../../../rippled/lib/convertRippleDate'
+import { localizeDate } from '../../../utils'
+import { DATE_OPTIONS } from '../../../transactionUtils'
+import { useLanguage } from '../../../hooks'
+
+export const Simple: TransactionSimpleComponent = ({
+  data,
+}: TransactionSimpleProps) => {
+  const { t } = useTranslation()
+  const language = useLanguage()
+  const {
+    oracleDocumentID,
+    provider,
+    uri,
+    lastUpdateTime,
+    assetClass,
+    priceDataSeries,
+  } = data.instructions
+  return (
+    <>
+      {oracleDocumentID !== undefined && (
```

Comment:
```suggestion
      {oracleDocumentID != null && (
```
It handles a few other cases in that situation.